src/main/java/io/leonis/zosma/ipc/peripheral/XboxController.java

```diff
@@ -0,0 +1,32 @@
+package io.leonis.zosma.ipc.peripheral;
+
+import io.leonis.zosma.game.Identity;
+import io.leonis.zosma.ipc.peripheral.XboxController.XboxControllerIdentity;
+import io.leonis.zosma.ipc.peripheral.component.*;
+import lombok.Value;
+
+/**
+ * The Class XboxController.
+ *
+ * Describes the state of the buttons in a xbox controller.
+ *
+ * TODO: This should be moved to Subra
```

Comment:
really? should it not go to frosty?

Reply:
I would like to postpone the move to frosty until that's ready. Probably we'll use a similar controller-interface there though.

As far as the move to subra; either I move the `ControllerPublisher` impl to `zosma` (along with the Jamepad-dependency) and map jamepads controllerstate to our `XboxController`-class in zosma, or i leave the mapping, dependency and `XboxController`-class in `subra`.

Do you have a strong preference on either option??

Reply:
i think it's too generic to keep in subra, so having it in zosma for now is imo a better option

Reply:
I agree; I'll append a `ControllerPublisher` and `ControllerDeducer` to this PR, and refactor everything in subra to just the `SSLControllerHandler`.